src/functionalTest/resources/features/F-001 - Create IA Staff Role Assignments/F-001.feature

```diff
@@ -116,3 +116,101 @@ Feature: F-001 : Create Role Assignments for Caseworker Users
     Then a positive response is received,
     And the response has all other details as expected
     And a successful call [to delete existing role assignments corresponding to the test actorId] as in [DeleteDataForRoleAssignments].
+
```

Comment:
Can you add a`task-supervisor"` and a`"case-allocator"` version to all new caseworker FTAs: including the existing `@S-001.03`, `@S-001.04` and `@S-001.05`.